apps/fortuna/src/api/revelation.rs

```diff
@@ -40,10 +40,6 @@ pub async fn revelation(
         })
         .inc();
 
-    let sequence: u64 = sequence
-        .try_into()
-        .map_err(|_| RestError::InvalidSequenceNumber)?;
```

Comment:
Are you sure the intention here was not to check if `try_into()` has some errors?

Reply:
`sequence` is already a `u64` so `try_into()` never fails.